source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -238,13 +245,17 @@ The following example performs these actions:
 
```

Comment:
S: shorten this code example by removing the newlines above when instantiating the FindOptions